docs/source/algorithms/MaxEnt-v1.rst

```diff
@@ -100,33 +100,33 @@ the output workspaces whether the algorithm was evolving towards the correct sol
 On the other hand, the user must always check the validity of the solution by inspecting *EvolChi* and *EvolAngle*,
 whose values will be set to zero once the true maximum entropy solution is found.
 
-.. table:: Table 1. Output workspaces for a real input workspace with M histograms and N bins
+.. table:: Table 1. Output workspaces for a real input workspace with M histograms and N bins, taking J iterations.
 
     +-------------------+------------------------------+----------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
     | Workspace         | Number of histograms         | Number of bins | Description                                                                                                                                                                                                                                                                                                        |
     +===================+==============================+================+====================================================================================================================================================================================================================================================================================================================+
-    | EvolChi           | M                            | MaxIterations  | Evolution of :math:`\chi^2` until the solution is found. Then all values are set to zero.                                                                                                                                                                                                                          |
+    | EvolChi           | M                            | J              | For spectrum :math:`s` in the input workspace, evolution of :math:`\chi^2` until the solution is found                                                                                                                                                                                                             |
```

Comment:
can we use a different letter for the spectrum? S is the entropy and I think it might be confusing

Reply:
Perhaps, that is why 's' is used for spectrum rather than 'S'.

Reply:
to avoid any confusion you could say the jth spectrum